sql/hive/src/test/scala/org/apache/spark/sql/hive/client/VersionsSuite.scala

```diff
@@ -237,6 +237,10 @@ class VersionsSuite extends SparkFunSuite with Logging {
       assert(client.getTableOption("default", "src").isDefined)
     }
 
+    test(s"$version: getTablesByName") {
+      assert(client.getTablesByName("default", Seq("src")).nonEmpty)
```

Comment:
This is just a special case. We need more to ensure all the hive metastore versions can correctly support it . Also negative cases are needed. :-)

Reply:
Sorry, what means? Hasn’t it been covered from 0.12 to 3.1?

Reply:
this only covers a single table. How about multiple tables? or mixed with non-existent tables? with illegal table names? or an empty seq? and so on. :-)